src/types/common.ts

```diff
@@ -1,5 +1,6 @@
 export type MapType = {
-    [key: string]: unknown;
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    [key: string]: any;
```

Comment:
Why any?

Reply:
`unknown` was broken with higher code, you can take a look